core/src/main/scala/org/apache/spark/util/SizeEstimator.scala

```diff
@@ -131,8 +131,9 @@ object SizeEstimator extends Logging {
       return System.getProperty(TEST_USE_COMPRESSED_OOPS_KEY).toBoolean
     }
 
-    // java.vm.info provides compressed ref info for IBM JDKs
-    if (System.getProperty("java.vendor").contains("IBM")) {
+    // java.vm.info provides compressed ref info for IBM and OpenJ9 JDKs
+    if (System.getProperty("java.vendor").contains("IBM") ||
```

Comment:
It seems that "IBM" and "OpenJ9" have similar information about compression.

Reply:
You are right, as described [here](https://github.com/eclipse/openj9/)
> The original source contribution to OpenJ9 came from the IBM "J9" JVM which has been used in production by thousands of Java applications for the last two decades.

Reply:
If so, it would be great if you mention explicitly the reason for the following functions.
-  `inferDefaultMemory` is not following `IBM` way.
- `getKrb5LoginModuleName` is not following `IBM` way.

Currently, the following statement in the PR description seems to give us the opposite idea.
```
In other places where see System.getProperty("java.vendor"), we can leave alone.
```

Thasnk!